### PR TITLE
Patch 2.0.2 Release

### PR DESCRIPTION
**Pull request by esreli**

### CODE REVIEW

**Comment by mhdostal** [comment]
@esreli I've updated the submodule to point to the Toolkit's 100.9 release and added the "git" doc to the 'Get Started' section (for issue #108).  Please take a look.

File: README.md [hunk 1]
```diff
@@ -50,12 +50,22 @@ Read the [docs](./docs/README.md) for a detailed explanation of the application,
 ## Get Started
 You will need [Xcode 11](https://itunes.apple.com/us/app/xcode/id497799835?mt=12) with Swift 5 and the [ArcGIS Runtime SDK for iOS](https://developers.arcgis.com/ios/latest/swift/guide/install.htm) (v100.1 or later) installed locally.
 
+The *Mapbook iOS app* now incorporates the [ArcGIS Runtime Toolkit for iOS](https://github.com/Esri/arcgis-runtime-toolkit-ios) (v100.9 or later) for additional functionality.
```
Comment (esreli) [inline README.md:53]:
```suggestion
*Mapbook for iOS* now incorporates the [ArcGIS Runtime Toolkit for iOS](https://github.com/Esri/arcgis-runtime-toolkit-ios) (v100.9 or later) for additional functionality.
```

**Comment by mhdostal** [comment]
@esreli - I added a release not about 100.9.0 SDK certification.  Please review, then I believe we're good-to-go!

**Comment by esreli** [comment]
@mhdostal I think we're good to go here! Would you mind merging this branch and cutting a release?

**Pull request merged**